src/components/Table/TableFormatters/AccountConnectFormatter.vue

```diff
@@ -67,6 +77,9 @@ export default {
       return this.titleText || this.$t('SelectProtocol')
     }
   },
+  mounted() {
+    this.hasPerm = this.formatterArgs.can()
+  },
   methods: {
     handleCommand(protocol) {
       if (protocol === 'Title') return
```

Comment:
Please note that there are several minor syntax errors on the provided template code. To optimize readability:

- Replace `@` with `{`
```js
<template>
</template>
```

Instead of wrapping all buttons within an `<el-button>` wrapper, use direct elements like `<button>`, which is more readable.

The `<if>` element can be replaced with simple conditionals for better understanding.
```js
<v-if="...condition">
```
Additionally, add some line breaks to make the code easier to read and understand.
```html
<div class="layout--wrap">

```

Ensure that the `<div>` tag contains its own content, as it's nested inside another element (`<template>` or similar).
This approach could result in a slightly cleaner and easier-to-navigate template.